assets/src/dashboard/components/storyAnimation/stories/index.js

```diff
@@ -95,3 +99,147 @@ export function _default() {
     </StoryAnimation.Provider>
   );
 }
+
+export function AMPStory() {
+  const pages = [
+    {
+      id: 'first-page',
+      animations: [
+        {
+          id: 'ir',
+          targets: ['el1'],
+          type: ANIMATION_TYPES.BOUNCE,
+          duration: 1000,
+        },
+      ],
+    },
+    {
+      id: 'second-page',
+      animations: [
+        {
+          id: 'a5',
+          targets: ['el2'],
+          type: ANIMATION_TYPES.BOUNCE,
+          duration: 2000,
+        },
+      ],
+    },
+    {
+      id: 'third-page',
+      animations: [
+        { id: 'a1', targets: ['el3'], type: ANIMATION_TYPES.BOUNCE },
+        {
+          id: 'a2',
+          targets: ['el4'],
+          type: ANIMATION_TYPES.BOUNCE,
+          delay: 100,
+        },
+        {
+          id: 'a3',
+          targets: ['el5'],
+          type: ANIMATION_TYPES.BOUNCE,
+          delay: 300,
+        },
+        {
+          id: 'a4',
+          targets: ['el6'],
+          type: ANIMATION_TYPES.BOUNCE,
+          delay: 500,
+        },
+      ],
+      elements: [
+        { id: 'el3', color: 'red', width: '50px' },
+        { id: 'el4', color: 'orange', width: '100px' },
+        { id: 'el5', color: 'blue', width: '200px' },
+        { id: 'el6', color: 'green', width: '150px' },
+      ],
+    },
+  ];
+
+  return (
+    <div style={{ width: '100%', height: '640px' }}>
+      <amp-story
+        standalone
+        title="My Story"
+        publisher="The AMP Team"
+        publisher-logo-src="https://example.com/logo/1x1.png"
+        poster-portrait-src="https://example.com/my-story/poster/3x4.jpg"
+      >
+        <amp-story-page id={pages[0].id}>
+          <StoryAnimation.Provider animations={pages[0].animations}>
+            <StoryAnimation.AMPKeyframes pageId={pages[0].id} />
+            <StoryAnimation.AMPAnimators pageId={pages[0].id} />
+
+            <StoryAnimation.AMPWrapper
+              pageId={pages[0].id}
+              target="el1"
+              style={{ width: '300px', height: '300px' }}
+            >
+              <amp-img
+                src="https://i.picsum.photos/id/1069/300/300.jpg"
+                width="300"
+                height="300"
+              />
+            </StoryAnimation.AMPWrapper>
+          </StoryAnimation.Provider>
+        </amp-story-page>
+        <amp-story-page id={pages[1].id}>
+          <StoryAnimation.Provider animations={pages[1].animations}>
+            <StoryAnimation.AMPKeyframes pageId={pages[1].id} />
+            <StoryAnimation.AMPAnimators pageId={pages[1].id} />
```

Comment:
`AMPKeyframes` and `AMPAnimators` need to be directly nested under `amp-story-page` but the `AMPWrapper` can be further nested under whatever other children.